packages/gatsby-theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -42,69 +35,9 @@ export const SideNavLayout = ({
   // Put queries for Top and Side navs here for performance
   // We should consider passing down the `sitePlugin` data in pageContext
   // rather than fetching the GraphQL here
-  const data = useStaticQuery(graphql`
-  {
-    site {
-      siteMetadata {
-        title
-      }
-    }
-    prInfo: envVars(name: { eq: "PR_INFO" }) {
-      num
-      url
-    }
-    allSitePage(filter: { context: { navSection: { ne: null } } },
-                sort: { fields: context___title }) {
-      nodes {
-        path
-        context {
-          title
-          navSection
-          source
-        }
-      }
-    }
-    sitePlugin(name: { eq: "gatsby-theme-patternfly-org" }) {
-      pluginOptions {
-        context
-        topNavItems {
-          text
-          path
-          contexts
-        }
-        sideNav {
-          core {
-            section
-            text
-            path
-          }
-          react {
-            section
-            text
-            path
-          }
-          get_started {
-            section
-            text
-            path
-          }
-          design_guidelines {
-            section
-            text
-            path
-          }
-          contribute {
-            section
-            text
-            path
-          }
-        }
-      }
-    }
-  }
-  `);
-  const { title } = data.site.siteMetadata;
-  const { num, url } = data.prInfo;
+  const data = {};
+  const { title } = 'title';
```

Comment:
I don't think this does anything - can we just assign title directly instead of destructuring?

Reply:
Whoops, you're correct! I'll fix this.

Eventually these guys will be props.

Reply:
Fixed!